gatsby-plugin-webfonts/src/gatsby-node.js

```diff
@@ -6,24 +6,35 @@ import createOptions from "./create-options";
 import webFonts from "./web-fonts";
 
 export const onPreBootstrap = async (
-  { cache, createContentDigest, store, pathPrefix },
+  { cache, createContentDigest, store, pathPrefix, reporter },
   pluginOptions,
 ) => {
   if (isEmpty(pluginOptions.fonts)) return;
 
   const { directory } = store.getState().program;
 
   const cacheFolder = path.join(directory, `.cache`, `webfonts`);
-  const publicFolder = path.join(directory, `public`, `static`, `webfonts`);
-
-  const options = createOptions({ ...pluginOptions, cacheFolder, pathPrefix });
+  const publicFolder = path.join(
+    directory,
+    `public`,
+    pathPrefix,
```

Comment:
the pathprefix does not have to be specified here
```
const publicFolder = path.join(directory, `public`, `static`, `webfonts`);
```

Reply:
Done